Specs/DateTime/Japanese/TimeParser.json

```diff
@@ -875,57 +835,33 @@
           }
         },
         "Start": 0,
-        "Length": 3
+        "Length": 4
       }
     ]
   },
   {
     "Input": "午後3時ごろ",
-    "NotSupported": "dotnet",
     "NotSupportedByDesign": "javascript,python,java",
     "Results": [
       {
         "Text": "午後3時ごろ",
```

Comment:
Shouldn't this one have an "approx" mod?

Reply:
The mod is added in Merged not in Time (same as in English).